src/lib/components/InputChip/InputChip.svelte

```diff
@@ -67,6 +67,27 @@
 		return { val: val, id: Math.random() };
 	});
 
+	function resetFormHandler() {
+		value = [];
```

Comment:
I'm sorry to say this but since InputChips can have an initial value, resetting to an empty array feels like the wrong choice in my opinion.

Reply:
That's true but input's can also have default value's like ``<input type="text" "value="initvalue" />`` and on form submissions those get cleared aswell, so I'm not sure what would be best, maybe even expose a prop that sets this boolean (resetToDefault, true resets to default value, false resets to no value)

Reply:
Didn't think of that, but yeah, good point.
No value might be the best choice then and if people want different behaviour they'll just have to use ``use:enhance``.
Wouldn't want to overcomplicate things!

Reply:
> Didn't think of that, but yeah, good point. No value might be the best choice then and if people want different behaviour they'll just have to use `use:enhance`. Wouldn't want to overcomplicate things!

I was about to say, the developer can always repopulate the InputChip programmatically if they wish to do so